test/ibmq/test_ibmq_client_v2.py

```diff
@@ -169,6 +169,21 @@ def test_get_job_includes(self, qe_token, qe_url):
         job_excluded = api.get_job(job_id, exclude_fields=['shots'])
         self.assertNotIn('shots', job_excluded)
 
+    @requires_qe_access
+    @requires_new_api_auth
+    def test_exception_message(self, qe_token, qe_url):
+        """Check exception has proper message."""
+        api = self._get_client(qe_token, qe_url)
+
+        with self.assertRaises(RequestsApiError):
```

Comment:
Can you revamp this block a bit? In general we try to make use of the context: You can actually make use of the "context":

```python
with self.assertRaises(X) as context_manager:
   do_something_that_raises_X

self.assertIn(context_manager.exception...., ...)
```

which helps a bit with readability, as we can avoid some nesting, indenting and re-raising.